src/renderer/components/layout/page-layout.tsx

```diff
@@ -80,11 +80,8 @@ export class PageLayout extends React.Component<PageLayoutProps> {
             <RecursiveTreeView data={navigation}/>
           </nav>
         )}
-        <div
-          className={cssNames("content", contentClass)}
-          id="ScrollSpyRoot"
-        >
-          <div className={cssNames(contentGaps && "flex column gaps")}>
+        <div className="content-wrapper" id="ScrollSpyRoot">
```

Comment:
Why do we have `id="ScrollSpyRoot"` instead of `className`? Is it intended to be single renderer element on the screen?

Reply:
We have to point ScrollSpy component which element will be used as `root`. Root is used as a target for intersection and mutation observers. So yes, we need only 1 such element, and it should be rendered once.